crystal/grip/shard.yml

```
@@ -11,6 +11,6 @@ targets:
 dependencies:
   grip:
     github: grip-framework/grip
-    version: ~> 1.0.0
+    version: 2.0.0
```

in order to use last patch version, should we not do ?

```suggestion
    version: ~> 2.0.0
```